pages/popup.css

```diff
@@ -0,0 +1,14 @@
+body {
+	background-color: #FFF;
+	margin: 0px;
+	padding: 0px;
+	width: 130px;
+}
+
+* {
+	-webkit-user-select: none;
```

Comment:
Haha, No, I just hate accidentally selecting stuff on accident, especially if there was no reason to select stuff in the first place. (I have a strange tendency to click randomly when I'm thinking.)